openapi-cli/src/test/resources/generators/client/utils/swagger/multipart_formdata_custom.yaml

```diff
@@ -0,0 +1,50 @@
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: OpenApi Petstore
+  license:
+    name: MIT
+servers:
+  - url: http://petstore.{host}.io/v1
+    description: The production API server
+paths:
+  /pets:
+    post:
+      summary: Create a pet
+      operationId: createPet
+      requestBody:
+        description: Pet
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                id:
+                  type: string
+                  format: uuid
+                  description: ID
+                address:
+                  type: object
+                  description: Address
+                  properties:
+                    street:
+                      type: string
+                      description: Street
+                    city:
+                      type: string
+                      description: City
+                profileImage:
+                  type: string
+                  format: binary
+                  description: Image
+            encoding:
+              profileImage:
+                contentType: image/png, image/jpeg
+                headers:
+                  X-Custom-Header:
```

Comment:
This file is named as `multipart_formdata_custom` because of this header? Shall we add a comment in top of the file stating this file contains a post request definition with multipart form data with a custom header?

Reply:
Added in https://github.com/ballerina-platform/openapi-tools/commit/b073cda18ba34b0a9e050f7cc820da3107ffa33b